content/learn/courses/anchor-positioning.md

```diff
@@ -0,0 +1,278 @@
+---
+eleventyExcludeFromCollections: true
```

Comment:
Remove this before merge, if we're ready to launch? Otherwise remove the link from the CSS Anchor Positioning talk page.